src/schedule/schedule.11tydata.js

```diff
@@ -0,0 +1,45 @@
+const { google, outlook, office365, yahoo, ics } = require('calendar-link');
+const simpleIcons = require('simple-icons');
+const { isBefore } = require('date-fns');
+
+module.exports = {
+	eleventyComputed: {
+		addToCalendarList: (data) => {
```

Comment:
I'd typically never nit on something like this, but this might be a good point for destructuring. You'd _mildly_ improve readability by just moving data.date->start up top.
```suggestion
		addToCalendarList: ({ title, description, date: start })) => {
```

Reply:
yeah. i was totally torn on this.
because i kinda liked having the separation.
but doing a destructured rename of `date: start` has me sold.
i'm in!